### DISCUSSION TITLE
How to integrate Webhook for event to get updates on discord

### DISCUSSION THREAD

**Discussion by Sachin-chaurasiya** [opencollective/opencollective#6437] [Q&A] [answered]
I have created an event under my collective and want to get the updates of that event in my discord channel, but when I post the update I'm not getting the same on my discord channel, can someone please guide me in the right direction?

Thanks.

**Comment by SudharakaP** [comment, marked as answer]
I've checked and there seems to be some issues related to this. I am working on a fix; https://github.com/opencollective/opencollective/issues/6483

**Reply by Sachin-chaurasiya** [reply]
Thanks for responding @SudharakaP.

**Reply by SudharakaP** [reply]
@Sachin-chaurasiya : Just to let you know the issue opened is fixed and deployed. So your problem should be solved now. Please try again. 👍🏽

**Reply by Sachin-chaurasiya** [reply]
Thanks, @SudharakaP, it's working now.
